parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetOutputFormat.java

```diff
@@ -317,6 +319,16 @@ private static int getMaxPaddingSize(Configuration conf) {
     return conf.getInt(MAX_PADDING_BYTES, DEFAULT_MAX_PADDING_SIZE);
   }
 
+  public static ValuesWriterFactory getValuesWriterFactory(Configuration conf) {
+    ValuesWriterFactory factory = ParquetProperties.DEFAULT_VALUES_WRITER_FACTORY;
+
+    if (factory instanceof Configurable) {
+      Configurable configurableFactory = (Configurable) factory;
+      configurableFactory.setConf(conf);
+    }
```

Comment:
1) since DefaultValuesWriterFactory does not implement Configurable maybe we just remove this if statement?
2) my question on configuration was how you decide which ValuesWriterFactory to use.